test/python/algorithms/test_grover.py

```diff
@@ -52,25 +52,27 @@ def test_groverop_getter(self, kind):
 
         self.assertEqual(Operator(expected), Operator(problem.grover_operator))
 
-    @data('list', 'statevector', 'callable')
+    @data('list_str', 'list_int', 'statevector', 'callable')
     def test_is_good_state(self, kind):
         """Test is_good_state works on different input types."""
-        if kind == 'list':
-            is_good_state = ['00', '11']
+        if kind == 'list_str':
```

Comment:
Is there a reason for changing the good states?

Reply:
Just want to use one good state for testing all the four types.   List[int] specifies the bitstr indices where the bits are set for the good state(s).   For instance, [0, 1] in List[int] is equivalent to ['11'] in List[str],   while [1] in List[int] is equivalent to ['01', '11'] in List[str].    And no List[int] representation for ['00', '11'].

Reply:
Ah I see! Good idea 👍🏻